src/sotarviclient.cc

```diff
@@ -37,7 +37,7 @@ void callbackWrapper(int fd, void *service_data, const char *parameters) {
 SotaRVIClient::SotaRVIClient(const Config &config_in, event::Channel *events_channel_in)
     : config(config_in), events_channel(events_channel_in), stop(false) {
   std::string client_config(config.rvi.client_config);
-  rvi = rviInitLogs(const_cast<char *>(client_config.c_str()), loggerGetSeverity() == LVL_trace);
+  rvi = rviInit(const_cast<char *>(client_config.c_str()));
```

Comment:
This was the change we make to lib_rvi to support logging can we keep this feature, please?